src/troute-network/troute/hyfeature_preprocess.py

```diff
@@ -379,9 +379,26 @@ def hyfeature_hybrid_routing_preprocess(
     break_network_at_waterbodies = waterbody_parameters.get(
         "break_network_at_waterbodies", False
     )
-    
+
     # if streamflow DA, then break network at gages
     break_network_at_gages = False
+
+    # temporary code to build diffusive_domain using given IDs of head segment and tailwater segment of mainstem.
+    '''
+    import pdb; pdb.set_trace()
+    headlink_mainstem = 242
+    twlink_mainstem = 160    
+    uslink_mainstem = headlink_mainstem
+    dslink_mainstem = 1 # initial value
+    mainstem_list =[headlink_mainstem]
+    while dslink_mainstem != twlink_mainstem:
+        dslink_mainstem = connections[uslink_mainstem][0]
+        mainstem_list.append(dslink_mainstem)
+        uslink_mainstem = dslink_mainstem   
+    diffusive_domain={}
+    diffusive_domain[twlink_mainstem] = mainstem_list
+    import pdb; pdb.set_trace()
```

Comment:
remove the two pdb.set_trace() calls here

Reply:
Removed. Thanks!